### ISSUE TITLE
Distortion in the test image results

### ISSUE THREAD

**Issue by andrepm06** [garris/BackstopJS#771]
Hello,

I've been facing some trouble while testing an part of my application. The image from the test is displayed in the report like this:

![failed_diff_test_visual__modal_exit_0_data-testconfirmationmodal_0_desktop](https://user-images.githubusercontent.com/6975975/39999398-6a4a22ce-575f-11e8-8973-b1b6f8a8e542.png)

Any idea of what's is going on?

**Comment by garris** [comment]
Can you post a screenshot of all three images in the report?

**Comment by garris** [comment]
Here you can see that the images are different sizes.  The distortion comes from the comparison algorithm not knowing what to do with that.

**Comment by garris** [comment]
Yes.  Totally known.  Backstop doesn’t support comparing tests where images are different sizes.

**Comment by andrepm06** [comment]
Ok then, thank you :-)
